examples/basic/05_error_handling.py

```
@@ -0,0 +1,271 @@
+    # 잘못된 API 키로 시도
+    print("1. 인증 에러 처리:")
+    try:
+        # 임시로 잘못된 API 키 설정
+        old_key = os.environ.get("OPENAI_API_KEY", "")
+        os.environ["OPENAI_API_KEY"] = "invalid-key"
+        
+        llm = LLM.create("gpt-4o-mini")
+        reply = llm.ask("안녕하세요")
+        
+    except AuthenticationError as e:
+        print(f"❌ 인증 실패: API 키를 확인해주세요.")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)